<?xml version="1.0"?>
<launch>
    <arg name="odom_topic" default="/visual_slam/odom"/>
    <arg name="drone_id" default="-1"/>
    <node name="fake_uwb_node" pkg="odom_correction" type="fake_uwb">
        <remap from="~my_odom" to="$(arg odom_topic)" />
        <param name="self_id" type="int" value="$(arg drone_id)"/>
        
    </node>

</launch>